### ISSUE TITLE
You can go to parameters/results page without specifying analytic

### ISSUE THREAD

**Issue by d47853** [gchq/gaffer-tools#750]
Currently you can go to the parameters or results page (using the nav bar) without first selecting which analytic you want to run. This means the user is greeted with an empty page which appears buggy.

To stop this, you could remove the nav bar to force users to go via selecting an analytic. If you wanted to go a step further, you could change the way the UI is routed

Where currently you use the javascript objects in the background and a flat routing layout ie:
- /analytics (show all the available analytics)
- /parameters (show the parameters for the analytic selected)
- /results (show the latest results for the analytic selected)

You could have a more analytic focussed routing system where you use the url to generate the parameters page and results. For example, if the analytic was called "my-analytic-name" it would look like:
- /analytics (show all the analytics)
- /my-analytic-name (show all parameters for my-analytic-name)
- /my-analytic-name/results (show latest results for my-analytic-name)

**Comment by g609bmsma** [comment]
How about instead of removing the tab/nav bar, we simply disable those links/tabs which have not been accessed yet, i.e. parameters can only be navigated to from the bar once an analytic has been selected, and results only once it has been executed? This then doesn't remove the backwards navigation later on in use. Implementing the router changes as well of course.

**Comment by d47853** [comment]
From a user experience point of view, I don't think having disabled links is as clear as only allowing one way of selecting an analytic.

**Comment by d47853** [comment]
Merged into develop
